phpcs.xml.dist

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<ruleset
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="vendor/squizlabs/php_codesniffer/phpcs.xsd"
+>
+    <arg name="cache" value=".phpcs-cache"/>
+
+    <rule ref="Doctrine"/>
```

Comment:
cdn77

Reply:
az to presunem

Reply:
? Nechápu

Reply:
az presunem pod cdn77 repo

Reply:
aha ... no nějak nechápu jak to s tím souvisí? :D Děláme to pro cdn, nemá smysl používat radom public standard, ale použít náš...

Reply:
neni random. ma lepsi odsazeni, tak ho pouzivam na veci, co kodim mimo, kdyz muzu. migrace na cdn standard bude jen zmena indentu vpodst.

Reply:
Hele asi to přežiju, ale přijde mi fakt zbytečný muset mentálně přepínat mezi dvěma styly ... stejně to děláme primárně pro CDN, tak mi přijde na hovno dělat to v jiném cs :D A máme tam sniffy navíc co Doctrine nemá.